### PR TITLE
Sort generated chain-specs to make them deterministic and recreate the hard-coded specs accordingly.

### PR DESCRIPTION
**Pull request by clangenb**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -75,7 +75,7 @@ jobs:
 
       - name: Release
         if: ${{ matrix.binary == 'release' }}
-        run: cargo build --release
+        run: cargo build --locked --release
```
Comment (clangenb) [inline .github/workflows/ci.yml:79]: `--locked` triggers a compiler error if the build would update the lock file.

File: scripts/dump_wasm_and_state_for_all_chains.sh [hunk 2]
```diff
@@ -9,9 +9,8 @@
 #
 # All arguments are optional.
 
-PARA_ID=${1:-2015}
```
Comment (clangenb) [inline scripts/dump_wasm_and_state_for_all_chains.sh:12]: As noted in the previous pr, the collators do no longer support this flag.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (brenzi) [reply .github/workflows/ci.yml:79]: good one

File: scripts/dump_wasm_and_state_for_all_chains.sh (hunk 2 above)
Comment (brenzi) [reply scripts/dump_wasm_and_state_for_all_chains.sh:12]: even better. one less source of inconsistency

**Review by brenzi — APPROVED** [review]

**Pull request merged**